### PR TITLE
Updating SpecvizExample notebook with new data link

### LINKED ISSUE #1758: Specviz example notebook data file no longer exists
**Reporter:** pllim

This does not work anymore.

```python
fn = "jw02732-c1001_t004_miri_ch1-longmediumshort-_x1d.fits"
result = Observations.download_file(f"mast:JWST/product/{fn}", local_path=f'{data_dir}/{fn}')
```

Need to find replacement that is not unlike what happened in https://github.com/spacetelescope/jdaviz/pull/1756 .

[🐱](https://jira.stsci.edu/browse/JDAT-2860)

 --- 

 DISCLAIMER: This issue was autocreated by the Jdaviz Issue Creation Bot on behalf of the reporter. If any information is incorrect, please contact [Duy Nguyen](https://github.com/duytnguyendtn)

### PR DESCRIPTION
**Pull request by megcdavis**
### Description

This pull request is to address a broken MAST download example link in the Specviz example notebook. The switch was made from downloading using the astroquery to astropy. This change can be temporary until the original notebook works again using astroquery. 

Fixes #1758

### Change log entry

- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a milestone set?
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

**Comment by camipacifici** [comment]
Dataset is linked from the STScI box.

**Review by kecnry — APPROVED** [review]
Thanks for the PR!  Just for context/bookkeeping: the MAST/astroquery downloads were added in #1680 and then reported broken in #1758.  If we can't quickly find a replacement file that we expect to be more stable, then this looks like a reasonable (temporary) solution to me.

Not sure if we should skip a changelog entry, add a new entry, or modify the one for #1680 here 🤔

**Review by pllim — CHANGES_REQUESTED** [review]
I might have a new MAST filename we can use that won't need such drastic diff.

File: notebooks/SpecvizExample.ipynb [hunk 1]
```diff
@@ -75,17 +76,9 @@
    "source": [
     "## Load a file to Specviz\n",
     "\n",
-    "Here we download a JWST spectrum from [MAST](https://masttest.stsci.edu/) using `astroquery`. \n",
-    "By default the downloaded file goes to your temp directory, and thus may eventually be \n",
-    "garbage collected (deleted) by your system. If you would like to have the file permanently, \n",
-    "simply uncomment the second line below and provide the directory path on your system where \n",
-    "you would like the file stored.\n",
+    "We will download an example JWST spectrum using `astropy`. The downloaded file can be found in the '/Users/username/.astropy/cache/download/' folder.\n",
```
Comment (bmorris3) [inline notebooks/SpecvizExample.ipynb:79]: Should we advertise the current default download destination path in `astropy.utils.data.download_file` as part of these docs? Maybe [a link](https://docs.astropy.org/en/stable/api/astropy.utils.data.download_file.html) to the docs on that function would be better.

File: notebooks/SpecvizExample.ipynb [hunk 2]
```diff
@@ -75,17 +76,9 @@
...
     "\n",
-    "One other thing to note about retrieving MAST data through astroquery is that it caches\n",
-    "the data by default. It is possible for files to be updated in MAST with more recent calibrations\n",
-    "but remain the same size, in which case your local cached file would not be replaced by the new\n",
-    "version. You can turn off caching by setting `cache=False` in the `download_file` call to\n",
-    "force it to re-download the file if desired."
+    "Data can also be downloaded from [MAST](https://masttest.stsci.edu/) using `astroquery`. Please see other notebooks for examples."
```
Comment (bmorris3) [inline notebooks/SpecvizExample.ipynb:81]: This is the "TEST VERSION" of MAST – don't we want https://mast.stsci.edu/ instead?

File: notebooks/SpecvizExample.ipynb [hunk 3]
```diff
@@ -20,6 +20,7 @@
     "import tempfile\n",
     "\n",
     "import astropy.units as u\n",
+    "from astropy.utils.data import download_file\n",
     "from astroquery.mast import Observations\n",
```
Comment (bmorris3) [inline notebooks/SpecvizExample.ipynb:24]: Is this line unnecessary now?

Comment (bmorris3) [reply notebooks/SpecvizExample.ipynb:24]:
```suggestion
    "\n",
```

**Comment by pllim** [comment]
Please see #1762 for a simpler fix.

**Comment by pllim** [comment]
Superseded by #1764

**Comment by pllim** [comment]
Thanks anyway!

**Pull request closed without merging**